py-polars/polars/internals/series.py

```diff
@@ -844,7 +844,7 @@ def median(self) -> float:
         """
         return self._s.median()
 
-    def quantile(self, quantile: float) -> float:
+    def quantile(self, quantile: float, interpolation: str = "nearest") -> float:
```

Comment:
Let's add a parameters sections which explains the possible interpolation strings